### ISSUE TITLE
Call to a member function getUuid() on a non-object

### ISSUE THREAD

**Issue by lolos** [rollbar/rollbar-php#232]
When we try to upgrade from version 0.18.2 to 1.2.0 (with the agent enabled), we get the error:

`FATAL: Call to a member function getUuid() on a non-object in /vendor/rollbar/rollbar/src/Rollbar.php:226`

When we disable the agent, everything works fine. Any ideas?

**Comment by cordoval** [comment]
Agent is python, so old version of rollbar maybe have put it in there and you are trying to send it with the new version of rollbar

```
self::log($level, $message, $extra_data) does not return a Response in your case, maybe false or 
```

something like
```
if (!$this->levelFactory->isValidLevel($level)) {
```
the level is invalid

after you upgrade for a long time, do you still get these? maybe flush your agent

if you don't have problems anymore you may close this issue

**Comment by rokob** [comment]
The AgentSender does not return a response so getUuid() gets called on nothing. Fixed by #239
